File: ncDados/dCheques.vb
Namespace nsCheques

  Public Class ColecaoCheques
    Inherits List(Of dCheques)
  End Class

  Public Class dCheques

    Private _cid As Integer
    Private _dataEmissao As Date
    Private _dataDeposito As Date
    Private _valor As Decimal
    Private _numero As String
    Private _vendaId As Integer
    Private _clienteId As Integer
    Private _baixado As String
    Private _bancoCodigo As String
    Private _bancoNome As String
    Private _agencia As String
    Private _conta As String

    Public Property cid() As Integer
      Get
        Return _cid
      End Get
      Set(ByVal value As Integer)
        _cid = value
      End Set
    End Property

    Public Property dataEmissao() As Date
      Get
        Return _dataEmissao
      End Get
      Set(ByVal value As Date)
        _dataEmissao = value
      End Set
    End Property

    Public Property dataDeposito() As Date
      Get
        Return _dataDeposito
      End Get
      Set(ByVal value As Date)
        _dataDeposito = value
      End Set
    End Property

    Public Property valor() As Decimal
      Get
        Return _valor
      End Get
      Set(ByVal value As Decimal)
        _valor = value
      End Set
    End Property

    Public Property Numero() As String
      Get
        Return _numero
      End Get
      Set(ByVal value As String)
        _numero = value
      End Set
    End Property

    Public Property vendasId() As Integer
      Get
        Return _vendaId
      End Get
      Set(ByVal value As Integer)
        _vendaId = value
      End Set
    End Property

    Public Property clienteId() As Integer
      Get
        Return _clienteId
      End Get
      Set(ByVal value As Integer)
        _clienteId = value
      End Set
    End Property

    Public Property baixado() As String
      Get
        Return _baixado
      End Get
      Set(ByVal value As String)
        _baixado = value
      End Set
    End Property

    Public Property bancoCodigo() As String
      Get
        Return _bancoCodigo
      End Get
      Set(ByVal value As String)
        _bancoCodigo = value
      End Set
    End Property

    Public Property bancoNome() As String
      Get
        Return _bancoNome
      End Get
      Set(ByVal value As String)
        _bancoNome = value
      End Set
    End Property

    Public Property agencia() As String
      Get
        Return _agencia
      End Get
      Set(ByVal value As String)
        _agencia = value
      End Set
    End Property

    Public Property conta() As String
      Get
        Return _conta
      End Get
      Set(ByVal value As String)
        _conta = value
      End Set
    End Property

  End Class

End Namespace

File: ncModelos/dCheques.cs
using System;
using System.Collections.Generic;

namespace ncModelos
{
    public class ColecaoCheques : List<dCheques>
    {
    }

    public class dCheques
    {
        private int _cid;
        private DateTime _dataEmissao;
        private DateTime _dataDeposito;
        private decimal _valor;
        private string _numero;
        private int _vendaId;
        private int _clienteId;
        private string _baixado;
        private string _bancoCodigo;
        private string _bancoNome;
        private string _agencia;
        private string _conta;

        public int cid
        {
            get => _cid;
            set => _cid = value;
        }

        public DateTime dataEmissao
        {
            get => _dataEmissao;
            set => _dataEmissao = value;
        }

        public DateTime dataDeposito
        {
            get => _dataDeposito;
            set => _dataDeposito = value;
        }

        public decimal valor
        {
            get => _valor;
            set => _valor = value;
        }

        public string Numero
        {
            get => _numero;
            set => _numero = value;
        }

        public int vendasId
        {
            get => _vendaId;
            set => _vendaId = value;
        }

        public int clienteId
        {
            get => _clienteId;
            set => _clienteId = value;
        }

        public string baixado
        {
            get => _baixado;
            set => _baixado = value;
        }

        public string bancoCodigo
        {
            get => _bancoCodigo;
            set => _bancoCodigo = value;
        }

        public string bancoNome
        {
            get => _bancoNome;
            set => _bancoNome = value;
        }

        public string agencia
        {
            get => _agencia;
            set => _agencia = value;
        }

        public string conta
        {
            get => _conta;
            set => _conta = value;
        }
    }

}
